.github/workflows/dispatch_submodule_update.yml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+name: Dispatch event for cortx submodule
+on:
+  push:
+    branches: [ main ]
```

Comment:
@venkuppu-chn, as we are raising this PR for `legacy-main` branch, should it be `legacy-main` as well?
CC - @shailesh-vaidya

Reply:
@gauravchaudhari02 I'm not getting you. Do we need to raise the same PR against legacy-main not main branch?

Reply:
![image](https://user-images.githubusercontent.com/61046500/150101449-5a1395db-9a65-45a3-9572-85087a4a6547.png)
You are raising PR towards `legacy-main` but dispatch event will trigger on `main` branch as per code.